### PR TITLE
Fix hooks with no args

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

**Comment by dlespiau** [comment]
lgtm

File: src/process.c [hunk 1]
```diff
@@ -61,7 +61,7 @@
 #include "netlink.h"
 
 static GMainLoop* main_loop = NULL;
-static GMainLoop* hook_loop = NULL;
+private GMainLoop* hook_loop = NULL;
```
Comment (grahamwhaley) [inline src/process.c]: heh, OK, had to go look up where you got 'private' from ;-) - had shudders of C++ - but see that it comes from common.h and changes meaning depending on UNIT_TESTING :-)

**Comment by grahamwhaley** [comment]
lgtm

**Comment by chavafg** [comment]
I opened an issue since docker cannot be launched using COR after this merge. 
https://github.com/01org/cc-oci-runtime/issues/221

**Comment by jodh-intel** [comment]
Fix raised on PR #223.

**Pull request merged**